bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -21,11 +21,12 @@
 from google.cloud._helpers import UTC
 from google.cloud._helpers import _date_from_iso8601_date
 from google.cloud._helpers import _datetime_from_microseconds
-from google.cloud._helpers import _datetime_to_rfc3339
 from google.cloud._helpers import _RFC3339_NO_FRACTION
 from google.cloud._helpers import _time_from_iso8601_time_naive
 from google.cloud._helpers import _to_bytes
 
+_RFC3339_MICROS_NO_ZULU = '%Y-%m-%dT%H:%M:%S.%f'
```

Comment:
For now, only bigquery uses this format, and we've gotten a lot more conservative about changes to core, so I think I'll hold off.